app/api/dao/task.py

```diff
@@ -6,9 +6,24 @@
 
 
 class TaskDAO:
+    """Data Access Object for Task functionalities."""
 
     @staticmethod
     def create_task(user_id, mentorship_relation_id, data):
+        """Creates a new task.
+
+        Creates a new task in a mentorship relation if the specified user is already involved in it.
+        Otherwise returns a message.
```

Comment:
remove this line, since this will always return a message, and you will indicate its meaning in the `Return:` section